src/xai_grok/errors/invalid_request_error.py

```diff
@@ -0,0 +1,4 @@
+class InvalidRequestError(Exception):
+
+    def __init__(self, message: str):
+        super().__init__(message)
```

Comment:
The indentation in the file is incorrect. The `__init__` method should be indented to be part of the `InvalidRequestError` class, otherwise it will be considered a standalone function.